samples/BenchmarkDotNet.Samples/BenchmarkDotNet.Samples.csproj

```diff
@@ -11,6 +11,7 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugSymbols>true</DebugSymbols>
     <NoWarn>$(NoWarn);CA1018;CA5351;CA1825</NoWarn>
+    <GenerateProgramFile>false</GenerateProgramFile>
```

Comment:
This is no longer needed since you added it in the test adapter props.

Reply:
The ability to copy across the `.props` file so that this property is not needed only happens via `PackageReference`, not `ProjectReference`. I have added a comment now to explain this.